Adding steps on applying standard config via medic-conf (#5658)
medic/medic: README.md

Navigate your browser to [`http://localhost:5988/medic/login`](http://localhost:5988/medic/login).

> ### Working on local environment with real devices 

Reviewer: Trying to be more succinct with the title:
```suggestion
### Testing locally with devices 
```

> This process is relevant for viewing the Medic webapp on mobile devices when the api service is run on a developer machine running Webapp >v3.5.0. Webapp v3.5.0 relies on service workers, which require a valid HTTPS certificate to function. Follow these steps to make your developer build accessible from your android device at the trusted url created by ngrok.

Reviewer: Suggested edits for clarity with CHT naming, and some other minor grammatical changes:
 
```suggestion
Follow the steps below to use an Android device with a development build of your application. This process is relevant when running v3.5.0 or greater of the Community Health Application Framework since it relies on service workers, which requires a valid HTTPS certificate. These steps will make your developer build accessible from your Android device by giving it a trusted URL created by _ngrok_.
```

> 1. Create an ngrok account at https://ngrok.com/ 

Reviewer: ```suggestion
1. Create a ngrok account at https://ngrok.com/ 
```

## Configuring Medic

We ship with one "standard" configuration, which can be a useful basis to start with. It is located at [./config/standard](https://github.com/medic/medic/tree/master/config/standard). 

> This app is highly configurable and can be modified to suit your needs. Read the guide for developing community health applications if you would like to customize your application further.

Reviewer 2: I'd put the second paragraph here above the first. Start with the role of configuration and then talk about the single specific configuration.

We don't really "ship with" a configuration. Really it is just "in this repository". I remember finding it really confusing until I realised that the other configurations were in other repositories. So maybe useful to mention?

Maybe link to the "guide for developing community health applications" ?

Configuration is performed using [Medic Configurer](https://github.com/medic/medic-conf). `medic-conf` expects a particular structure (seen in the standard config above). It compiles forms and configuration into the required formats, as well as uploading that configuration and performing other tasks.

To import the standard configuration:

1. Install medic-conf: `npm install -g medic-conf`
2. Navigate to the configuration you want to import: `cd <medic-repo>/config/standard`
1. Ensure the app/api is running. Specifically on localhost for these instructions. 

> 3. Import the config: `medic-conf --url http://username:password@localhost:5984`

Reviewer 3: This should reference the api port, so the docs should probably say `5988` not `5984`.

Reviewer 2: As of medic-conf 2.0 this is `--url=http://username:password@localhost:5988`

Author: I've updated to use 5988. Looking at the code we replace 5984 with 5988.

Reviewer 2: I think you still need the `=` sign though. `--url=http://...`
